trl/trainer/dpo_config.py

```diff
@@ -125,14 +150,14 @@ class DPOConfig(TrainingArguments):
     truncation_mode: str = "keep_end"
     max_length: Optional[int] = None
     max_prompt_length: Optional[int] = None
-    max_target_length: Optional[int] = None
+    max_completion_length: Optional[int] = None
```

Comment:
I realized this was not consistent with other trainers. Changing it shouldn't be breaking, right? Or maybe add a post-init warning?
@kashif @edbeeching @lewtun

Reply:
I'll add a post-init warning.